test/data/Cardano/Wallet/fees

```diff
@@ -0,0 +1,115 @@
+[250000,250000]|[100000,100]|188298
+[250000,250000]|[100000,1000]|188298
+[250000,250000]|[100000,10000]|188298
+[120000,120000,120000]|[100000,1]|196252
+[120000,120000,120000]|[100000,10]|196252
+[120000,120000,120000]|[100000,100]|196428
+[120000,120000,120000]|[100000,1000]|196428
+[120000,120000,120000]|[100000,10000]|196428
+[90000,90000,90000,90000]|[100000,1]|204382
+[90000,90000,90000,90000]|[100000,10]|204382
+[90000,90000,90000,90000]|[100000,100]|204558
+[90000,90000,90000,90000]|[100000,1000]|204558
+[90000,90000,90000,90000]|[100000,10000]|204558
+[70000,70000,70000,70000,70000]|[100000,1]|212512
+[70000,70000,70000,70000,70000]|[100000,10]|212512
+[70000,70000,70000,70000,70000]|[100000,100]|212688
+[70000,70000,70000,70000,70000]|[100000,1000]|212688
+[70000,70000,70000,70000,70000]|[100000,10000]|212688
+[60000,60000,60000,60000,60000,60000]|[100000,1]|220642
+[60000,60000,60000,60000,60000,60000]|[100000,10]|220642
+[60000,60000,60000,60000,60000,60000]|[100000,100]|220818
+[60000,60000,60000,60000,60000,60000]|[100000,1000]|220818
+[60000,60000,60000,60000,60000,60000]|[100000,10000]|220818
+[51000,51000,51000,51000,51000,51000,51000]|[100000,1]|228772
+[51000,51000,51000,51000,51000,51000,51000]|[100000,10]|228772
+[51000,51000,51000,51000,51000,51000,51000]|[100000,100]|228948
+[51000,51000,51000,51000,51000,51000,51000]|[100000,1000]|228948
+[51000,51000,51000,51000,51000,51000,51000]|[100000,10000]|228948
+[100000,100000,100000]|[1,1,1]|203988
+[60000,60000,60000,60000]|[1,1,1]|212118
+[50000,50000,50000,50000,50000]|[1,1,1]|220248
+[60000,60000,60000,60000]|[100,1,1]|212381
+[60000,60000,60000,60000]|[100,1,100]|212380
+[60000,60000,60000,60000]|[100,100,100]|212379
+[50000,50000,50000,50000,50000]|[100,1,1]|220510
+[50000,50000,50000,50000,50000]|[100,100,1]|220510
+[50000,50000,50000,50000,50000]|[100,100,100]|220510
+[42000,42000,42000,42000,42000,42000]|[1,1,1]|228378
+[42000,42000,42000,42000,42000,42000]|[1,1,100]|228640
+[42000,42000,42000,42000,42000,42000]|[100,1,1]|228640
+[42000,42000,42000,42000,42000,42000]|[1,100,1]|228640
+[42000,42000,42000,42000,42000,42000]|[100,100,1]|228641
+[42000,42000,42000,42000,42000,42000]|[1,100,100]|228641
+[42000,42000,42000,42000,42000,42000]|[100,1,100]|228641
+[42000,42000,42000,42000,42000,42000]|[100,100,100]|228639
+[42000,42000,42000,42000,42000,52000]|[1,10000,1]|228640
+[42000,42000,42000,42000,52000,52000]|[1,10000,10000]|228641
+[52000,42000,42000,42000,42000,52000]|[10000,10000,100]|228640
+[62000,62000,62000,62000,62000,42000]|[100000,100,1]|228640
+[82000,82000,82000,82000,82000,42000]|[100000,100000,1]|228640
+[42000,42000,42000,42000,42000,42000,42000,42000,42000,42000,42000,42000,42000,42000,42000]|[100000,100000,100000]|302073
+[42000,42000,42000,42000,42000,42000,42000,42000,42000]|[100000,1000,100]|253294
+[42000,42000,42000,42000,42000,42000,42000,42000,42000]|[100000,10000,1]|253032
+[75000,75000,75000,60000]|[100,1,1]|212381
```

Comment:
I think it's a bit overkill here :/ What's the value of that many cases compared to a few well-crafted unit tests + properties ?

Reply:
I produced it in a quasi-random way just to see if fee is as expected by my when investigating old way of estimation. Will take representive examples from this file to `test/data/Cardano/Wallet/fees1`